srcpkgs/textadept/template

```diff
@@ -0,0 +1,82 @@
+revision=1
+wrksrc="textadept-textadept_${version}"
+build_wrksrc="src"
+hostmakedepends="autoconf automake libtool wget unzip pkg-config glib-devel ncurses"
+makedepends="gtk+-devel ncurses-devel"
+short_desc="Fast, minimalist, and extensible text editor for programmers"
+maintainer="reback00 <reback00@protonmail.com>"
+license="MIT"
+homepage="https://foicica.com/textadept/"
+distfiles="https://github.com/rgieseke/textadept/archive/textadept_${version}.tar.gz
+ http://www.lua.org/ftp/lua-5.3.5.tar.gz
+ http://www.inf.puc-rio.br/~roberto/lpeg/lpeg-1.0.2.tar.gz
+ http://github.com/keplerproject/luafilesystem/archive/v1_7_0_2.zip
+ http://invisible-mirror.net/archives/cdk/cdk-5.0-20150928.tgz
+ http://www.leonerd.org.uk/code/libtermkey/libtermkey-0.20.tar.gz
+ http://www.inf.puc-rio.br/~roberto/lpeg/lpeg-1.0.2.tar.gz
+ http://www.leonerd.org.uk/code/libtermkey/libtermkey-0.20.tar.gz"
+checksum="2cb2aa8266c166ed4a3f534843aa75cec0dc1ced09c007aaef540e3a7999870b
+ 0c2eed3f960446e1a3e4b9a1ca2f3ff893b6ce41942cf54d5dd59ab4b3b058ac
+ 48d66576051b6c78388faad09b70493093264588fcd0f258ddaab1cdd4a15ffe
+ 9174ab891f1d392b42a587b2afb737f9565ecf8a860d2e5a2bff457858f58478
+ 028da75d5f777a1c4184f88e34918bd273bd83bbe3c959bc11710c4f0ea2e448
+ 6c0d87c94ab9915e76ecd313baec08dedf3bd56de83743d9aa923a081935d2f5
+ 48d66576051b6c78388faad09b70493093264588fcd0f258ddaab1cdd4a15ffe
+ 6c0d87c94ab9915e76ecd313baec08dedf3bd56de83743d9aa923a081935d2f5"
+skip_extraction="
+ lua-5.3.5.tar.gz
+ lpeg-1.0.2.tar.gz
+ v1_7_0_2.zip
+ cdk-5.0-20150928.tgz
+ libtermkey-0.20.tar.gz
+ lpeg-1.0.2.tar.gz
+ libtermkey-0.20.tar.gz
+"
+
+# Take cached dependency files so that they are not downloaded by build
+# script again. Adapted from "libreoffice" template.
+post_fetch() {
+	local srcdistdir=${XBPS_SRCDISTDIR}/${pkgname}-${version} pkg
+	mkdir -p ${wrksrc}/src
+	for pkg in ${skip_extraction}; do
+		if [ "${pkg%.dll}" == "${pkg}" ]; then
+			ln -svf ${srcdistdir}/${pkg} ${wrksrc}/src
+		fi
+	done
+}
+
+pre_configure() {
+	# For cross builds
+	vsed -i -e 's/CC = /CC ?= /' -e 's/CFLAGS = /CFLAGS ?= /' -e 's/CXX = /CXX ?= /' -e 's/CXXFLAGS = /CXXFLAGS ?= /' Makefile
+	make deps
+}
+
+do_build() {
+	make ${makejobs}
+	make ${makejobs} curses
+}
+
+do_install() {
+	make PREFIX=/usr DESTDIR="${DESTDIR}" install
+	make curses PREFIX=/usr DESTDIR="${DESTDIR}" install
+
+	# Binaries in /usr/share/textadept are not allowed
+	# So we relocate to /usr/lib/textadept
+	mkdir -p "${DESTDIR}/usr/lib/"
+	mv "${DESTDIR}/usr/share/textadept" "${DESTDIR}/usr/lib/textadept"
+	ln -sf "/usr/lib/textadept/textadept" "${DESTDIR}/usr/bin/textadept"
+	ln -sf "/usr/lib/textadept/textadept-curses" "${DESTDIR}/usr/bin/textadept-curses"
+
+	# For icons
+	mkdir -p "${DESTDIR}/usr/share/pixmaps"
+	ln -s "/usr/lib/textadept/core/images/textadept.svg" "${DESTDIR}/usr/share/pixmaps/textadept.svg"
+	ln -s "/usr/lib/textadept/core/images/ta_48x48.png" \
+		"${DESTDIR}/usr/share/pixmaps/textadept.png"
+
+	# For license
```

Comment:
drop this comment, use of `vlicense` is self-explanatory

Reply:
Removed. (thanks to @domrc)